context/app/static/js/components/organ/Azimuth/Azimuth.jsx

```diff
@@ -1,17 +1,43 @@
 import React from 'react';
+import marked from 'marked';
 
-import Paper from '@material-ui/core/Paper';
-
+import Button from '@material-ui/core/Button';
 import SectionHeader from 'js/shared-styles/sections/SectionHeader';
 import SectionContainer from 'js/shared-styles/sections/SectionContainer';
+import VisualizationWrapper from 'js/components/Detail/visualization/VisualizationWrapper/VisualizationWrapper';
+import LabelledSectionText from 'js/shared-styles/sections/LabelledSectionText';
+import { SpacedSectionButtonRow } from 'js/shared-styles/sections/SectionButtonRow';
+
+import { StyledPaper } from './style';
 
 function Azimuth(props) {
-  const { azimuth } = props;
+  const { config } = props;
+  const datarefHtml = marked(config.dataref);
+  const demodataHtml = marked(config.demodata);
+  const detailsHtml = marked(config.details);
 
   return (
     <SectionContainer>
-      <SectionHeader>Description</SectionHeader>
-      <Paper>TODO: {JSON.stringify(azimuth)}</Paper>
+      TODO: Add info popover.
+      <SpacedSectionButtonRow
+        leftText={<SectionHeader>Reference-Based Analysis</SectionHeader>}
```

Comment:
`SectionHeader` comes with some margin baked in so that might misalign things, but no concern if it looks ok.

Reply:
Looks ok to me... but I don't have the best eye for this. There will definitely need to be a second (third, fourth, ...) pass.